admin/views/pupil-pin/generate-pins-overview.ejs

```diff
@@ -43,7 +53,11 @@
         <form action="/pupil-pin/generate-pins-list/lastName/asc" name="generate-pins-list">
             <div class="form-group">
                 <div class="form-buttons">
+                    <% if (error) { %>
+                    <input type="submit" class="button" disabled value="Generate PINs"/>
```

Comment:
Can you confirm that `disabled` works in IE8?  Or does it need `disabled="disabled"`?

Reply:
https://www.w3schools.com/tags/att_input_disabled.asp
IE>6.0 based on w3 schools.

Reply:
👍